tests/Integration/inc/ThirdParty/Plugins/Optimization/AMP/TestCase.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration\inc\ThirdParty\Plugins\Optimization\AMP;
+
+use WPMedia\PHPUnit\Integration\TestCase as BaseTestCase;
+
+abstract class TestCase extends BaseTestCase {
+	protected function setSettings( $option, $value ) {
+		\AMP_Options_Manager::update_option( $option, $value);
```

Comment:
Import `AMP_Options_Manager` instead of `\`.

Reply:
done in https://github.com/wp-media/wp-rocket/pull/2459/commits/17739b5ec01a9c81159965e0ccbbc297f367d950